cloud_utils/scheduler/kubernetes_connector.py

```diff
@@ -58,10 +58,25 @@ def _wait_for_job_completion(
         job = client.BatchV1Api().read_namespaced_job_status(
             **gamla.add_key_value("name", name)(options)
         )
+
         if job.status.succeeded:
             return
-        if job.status.failed:
-            raise Exception("Job failed")
+        if (
+            job.status.failed and job.status.failed > 1
```

Comment:
where is this retry defined ? Shouldn't we always output the logs ?

Reply:
We set the backoff limit to 1 here
https://github.com/hyroai/cloud-utils/blob/master/cloud_utils/scheduler/kubernetes_connector.py#L218

I think that getting the logs every 60 sec might be a bit too much and not very interesting.
Let's start with this (log only on 2nd failure) and add more if needed?